lib/chord.js

```diff
@@ -321,6 +321,30 @@ var identify = function () {
 
   var results = _.map(inversions, tryInversion);
```

Comment:
I'd prefer if you combined all the variables that come next using `_.chain()` and then just return here.